setup.py

```diff
@@ -30,7 +30,8 @@
           "boto3>=1.9.226,<2",
           "botocore>=1.12.226,<2",
           "deprecation>=2.0.7,<3",
-          "urllib3>=1.25.7,<2"
+          "urllib3>=1.25.7,<2",
+          "numpy>=1.19.5,<2"
```

Comment:
This might be tricky, current version of numpy requires using Python version >=3.7

Reply:
1.19.5 is compatible with python 3.6, so I think we're ok there